MichaelEWalker87/hogwarts-house-cup
# Testing

# Testing: 

## What's this PR do? 

* Set the unit testing for Gameplay, Pick_player, Play_Cards, Player_Amount, Quick_Start, Rules, SingleCard

* Does integration testing in APP



## Where should the reviewer start? 

* I would start with Player_Amount moving through the unit test as they appear in the application then end with App.test.js

## How should this be manually tested? 

* Run npm test in the root folder the tester should see 
Test Suites: 7 passed, 7 total
Tests:       25 passed, 25 total

## Any background context you want to provide? 

* I was only planning to unit test on this branch due to difficulties in Pick_Player I changed methods to be more efficient. 

## What are the relevant tickets? 
* Addresses:
* Closes: #19 #21 #17 #15 #14 #13 #12 #6


## Screenshots (if appropriate) 

* none at this time 

## Questions:

* I tested everything I could think to test the happy and sad path I was not sure is snapshot testing is needed due to test each component is already being tested in the unit and in the integration test
